bin/methods/create_a_teacher.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../classes/teacher'
+
+class CreateNewTeacher
+  def initialize(people)
+    @people = people
+  end
+
+  def creates_a_teacher
+    clear
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    new_teacher = Teacher.new(specialization, age, name)
+    @people << new_teacher
+    added_msg('Teacher')
+  end
```

Comment:
- [ ] good job cleaning up the main.rb file 👍 in this method text is printed in the console, information is gathered and it is processed, this is not in line with the single responsibility principle, try to split this into smaller methods with a single responsibility. for more details have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).